pkg/pgmodel/pgx_test.go

```diff
@@ -432,18 +415,15 @@ func TestPGXInserterInsertSeries(t *testing.T) {
 						}
 					}
 					return
-				case c.callbackErr != nil:
-					if err != c.callbackErr {
-						t.Errorf("unexpected callback error:\ngot\n%s\nwanted\n%s", err, c.callbackErr)
-					}
-					return
 				default:
 					t.Errorf("unexpected error: %v", err)
 				}
 			}
 
-			if calls != len(c.series) {
-				t.Errorf("Callback called wrong number of times: got %v expected %d", calls, len(c.series))
+			for _, si := range lsi {
+				if si.seriesID == 0 {
```

Comment:
we should also check `si.seriesID == -1` since that's the sentinel we use for not-yet-set